.gitpod.yml

```
@@ -0,0 +1,42 @@
+# Commands to start on workspace startup
+
+ports:
+  # when port 8000 becomes available open a browser preview
+  # of the running django pattern library instance
+  - port: 8000
+    onOpen: open-preview
+
+tasks:
+  # initialise and install the python part of the app by installing poetry
+  # and the python dependencies
+  - init: |
+      clear
+      pip install poetry
```

Could we change this to match the Docker setup?

```suggestion
      pip install --user "poetry>=1.1.12,<2"
```

---

I checked this out and found the `--user` flag isn't required because in Gitpod `PIP_USER` is set to `yes`

[PIP_USER](https://github.com/gitpod-io/gitpod/issues/1997#issuecomment-708480259)